packages/account/src/Sections/Security/ConnectedApps/__tests__/connected-apps.spec.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { render, screen, waitFor, waitForElementToBeRemoved, fireEvent } from '@testing-library/react';
+import ConnectedApps from '../connected-apps';
+import { act } from '@testing-library/react';
+
+const oauth_apps_list = [
+    {
+        name: 'Local',
+        app_markup_percentage: 0,
+        app_id: 9999,
+        scopes: ['read', 'admin', 'trade', 'payments'],
+        last_used: '2021-10-31 06:49:52',
+    },
+];
+
+jest.mock('@deriv/shared/src/services/ws-methods', () => ({
+    __esModule: true,
+    default: 'mockedDefaultExport',
+    WS: {
+        authorized: {
+            send: ({ oauth_apps, revoke_oauth_app }) => {
+                if (revoke_oauth_app) oauth_apps_list.pop();
```

Comment:
pop will change the original array as a result, which might result in tests failing in parallel. is it possible to change this value to a manual expectation? it adds more clarity to it as well.